ecms_base/modules/custom/ecms_health_file_migration/config/install/migrate_plus.migration.health_file_json_to_file.yml

```diff
@@ -7,27 +7,29 @@ destination:
   plugin: 'entity:file'
 
 source:
-  constants:
-    DRUPAL_FILE_DIRECTORY: 'public://'
-    DIRECTORY_SEPERATOR: '/'
```

Comment:
Why are these being deleted? They are being used in the `.module` file.

Reply:
The ticket says to update the source. One of the three files was already set to the indicated source. I replaced the source part of the other yml files to match the one that was already set. 

Part of the replacement was deleting these lines. because they were not part of the source that I was using as a template.